core/src/main/java/org/lflang/analyses/statespace/StateInfo.java

```diff
@@ -0,0 +1,24 @@
+/** A class that represents information in a step in a counterexample trace */
```

Comment:
Probably this comment belongs on line 6, above `public class StateInfo`?

> [A doc comment is written in HTML and must precede a class, field, constructor or method declaration](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html)

The same comment applies to `StateSpaceInfo.java`.